### PR TITLE
API naming advise about language and charset feedback in #108

### PR DESCRIPTION
**Pull request by atanassov**
Following up on issue https://github.com/w3ctag/design-principles/issues/108. We are adding prose for the language and character set to be used.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -137,6 +137,10 @@ and by name association.
 
 Please consult widely on names in your APIs.
 
+API naming *must* be done in easily readable US-English language and fall into the ASCII range. 
+For example `horizongalAlignment` is more English-readible name than `alignmentHorizontal`.
```
Comment (annevk) [inline index.bs]:
```suggestion
For example `horizontalAlignment` is a more English-readable name than `alignmentHorizontal`. 
```

**Comment by annevk** [comment]
I think the rebase failed unfortunately and now it's changing more than it should.

File: index.bs [hunk 2]
```diff
@@ -425,7 +432,61 @@ the API visibly relates to garbage collection details. By contrast, special APIs
 purpose such as `WeakRef` and `FinalizationGroup` have their GC interaction as a specific part of
 their contract with the developer.
 
-<h2 id="api-surface">JavaScript API Surface Concerns</h2>
+<h2 id="api-surface">API Surface Concerns</h2>
```
Comment (hober) [inline index.bs]: this heading rename shouldn't happen

File: index.bs [hunk 3]
```diff
@@ -425,7 +432,61 @@ the API visibly relates to garbage collection details. By contrast, special APIs
...
+
+<h3 id="naming-is-hard">Naming things</h3>
```
Comment (hober) [inline index.bs]: please don't duplicate this whole section

File: index.bs (hunk 2 above)
Comment (atanassov) [reply index.bs]: Addressed with the rebase.

File: index.bs (hunk 3 above)
Comment (atanassov) [reply index.bs]: Addressed with the rebase.

File: index.bs [hunk 4]
```diff
@@ -4,9 +4,9 @@ Group: W3C TAG
 Shortname: design-principles
 Repository: w3ctag/design-principles
 Status: DREAM
-Editor: Travis Leithead, Microsoft, travil@microsoft.com
```
Comment (hober) [inline index.bs]: Why is this change in here? Did something go wrong with the rebase?

Comment (atanassov) [reply index.bs]: Yeah, moved to Former Editor: - already part of master

Comment (hober) [reply index.bs]: If it's already in master, why is it showing up in this diff?

**Comment by annevk** [comment]
You also removed some of the earlier fixes I suggested.

File: index.bs (hunk 4 above)
Comment (atanassov) [reply index.bs]: Added the dropped 'a' from @annevk change and got rid of the editor's list change which is in master and not sure why it was flagged as diff here...

**Comment by dbaron** [comment]
With this kind of PR, it's better (next time) to squash rather than merge, so you don't have all of that messy history permanently in the repository.

**Comment by dbaron** [comment]
@atanassov force-pushed to revert from 68b84b366f4b1bf3dcbf1c746693f0d31ed8ec57 back to 82dea718c47a6f4a9e3dc9541a0c88a4cf55186e, so this needs to get re-landed.  Not sure how to get github to recognize that, though.

**Comment by atanassov** [comment]
I did manually re-land the squashed commit to master as b66b7f45c047adf2ad63f5902a875d1f1bb65f82 
Next, we should change this repo policy to have squash-merge as the default option :)

**Pull request merged**